### ISSUE TITLE
Audio Error in Samsung M52 5G

### ISSUE THREAD

**Issue by ReshiramZ** [Gamer64ytb/Lemonade#10]
my phone:M52 5G

before citra was closed,
there was a bug that bothers me, which is about the emulation audio,I remember that on log there was some audio problem, but I couldn't identify the problem .
(and the bug is still here)

The audio comes out almost nothing, but if I record my screen, it comes out a little more, but it's still buggy.

for now, this only happens with mario 3D land, the pokemon alpha sapphire the audio is perfect.
(log when I played Mario 3D Land)

[citra_log.txt](https://github.com/Gamer64ytb/Lemonade/files/14657290/citra_log.txt)
[](url)

**Comment by WreckageBrother** [comment]
Yeah exactly same problem on My Samsung Galaxy M62. Exynos 9825. 

[lemonade_log.txt](https://github.com/Lemonade-emu/Lemonade/files/14830260/lemonade_log.txt)

**Comment by Gamer64ytb** [comment]
hows audio actually for you? the same for alpha 2.1?

**Comment by ReshiramZ** [comment]
Sorry, but I changed my cell phone a few days ago, so I can't say if it has improved or not, because my current cell phone is playing the audio without any problems, I really apologize.

**Comment by WreckageBrother** [comment]
> hows audio actually for you? the same for alpha 2.1?

Unfortunately still no audio with the latest 2.1 update 
[lemonade_log.txt](https://github.com/Lemonade-emu/Lemonade/files/14903972/lemonade_log.txt)
